src/irmin/bheap.ml

```diff
@@ -0,0 +1,117 @@
+(**************************************************************************)
```

Comment:
Can you add the exact commit of bheap in the commit message? I will help for the maintenance of that piece of code.